### PR TITLE
Add extra spaces

### PR DESCRIPTION
**Pull request by newbazz**
Fixes https://github.com/publiclab/plots2/issues/6059

Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by newbazz** [comment]
image
This is how it looks after the changes

File: app/views/tag/show/_header.html.erb [hunk 1]
```diff
@@ -147,11 +147,11 @@
 .bottom-right {
   position: absolute;
   bottom: 10px;
-  right: 16px;
+  right: 2px;
```
Comment (CleverFool77) [inline app/views/tag/show/_header.html.erb]: As I suggested here https://github.com/publiclab/plots2/issues/6059#issuecomment-514707527 , add extra space instead removing them. Thanks !!

**Comment by CleverFool77** [comment]
Btw great work with constant PRs.
Thanks !!

File: app/views/tag/show/_header.html.erb (hunk 1 above)
Comment (newbazz) [reply app/views/tag/show/_header.html.erb]: I have added extra space

**Comment by newbazz** [comment]
image
here is the full screenshot @CleverFool77

**Comment by CleverFool77** [comment]
Sure !! One more screenshot without scrolling it horizontally. Right now it looks like its scrolled somewhat towards right.

**Comment by newbazz** [comment]
image

Here it is @CleverFool77

**Comment by jywarren** [comment]
Super!!!! Thanks so much!!!

**Pull request merged**